charts/datadog/values.yaml

```diff
@@ -821,7 +821,7 @@ datadog:
       # This should be set to true when using EKS, GKE or if containerd is configured to
       # discard uncompressed layers.
       # This feature will cause the SYS_ADMIN capability to be added to the Agent container.
```

Comment:
Could you update the description to explains the limitation if they move the option back to `false`: high SBOM scan error rate.

Reply:
Done